### ISSUE TITLE
sauvegarde des tags lors de la conversion MP3 -> OGG

### ISSUE THREAD

**Issue by trebmuh** [Houston4444/ActionMenuAudioConverter#3]
Salut Mathieu

Je suis en train de mettre un peu d'ordre dans ma bibliothèque musicale.
J'ai l'impression que lorsque l'on fait une conversion, les tags ne sont pas conservés.
Après vérification, si je convertis un OGG vers un MP3, les tags sont sauvegardés, mais si je fais l'inverse : MP3 -> OGG, ils ne le sont pas.
Ça serait chouette de pouvoir les conserver.

---

Résumé :
OGG -> MP3 : tags sauvegardés
_**MP3 -> OGG : tags pas sauvegardés**_
FLAC -> MP3 : tags sauvegardés

**Comment by Houston4444** [comment]
Salut Olivier.
Il faudrait regarder du côté de ffmpeg, c'est probablement un bug de ce côté. Ou si ce n'en est pas un, voir s'il existe un argument pour conserver les métadonnées.

**Comment by trebmuh** [comment]
Pour info, une conversion FLAC -> MP3 conserve les tags.

_Je mets à jour mon premier message en fonction_

**Comment by Houston4444** [comment]
Mmmh, je n'arrive pas à reproduire le bug, ici les metadonnées sont sauvegardées dans tous les cas.

**Comment by trebmuh** [comment]
Bizarre. T'es sur une LZK4 ?

**Comment by Houston4444** [comment]
Oui. Et je viens de vérifier, j'utilise bien la version empaquetée de ActionMenuAudioConverter

**Comment by trebmuh** [comment]
C'est vraiment bizarre ça. De quoi ça pourrait bien venir...?
Tu n'as pas d'autres dépôts que les Debian et les LZK ?

**Comment by Houston4444** [comment]
Mmmh, si ça se trouve on ne parle pas de la même chose...
Quand je convertis un mp3 en ogg, les métadonnées de la piste (nom de la chanson, artiste, album, année...) sont bien appliquées au fichier ogg.

Il est peut-être possible aussi que ça dépende de la façon dont le fichier source a été encodé, par exemple, ça pourrait fonctionner avec certains mp3 et pas avec d'autres.

ffmpeg vient bien de debian chez moi: 7:4.3.6-0+deb11u1

**Comment by trebmuh** [comment]
> Mmmh, si ça se trouve on ne parle pas de la même chose...
> Quand je convertis un mp3 en ogg, les métadonnées de la piste (nom de la chanson, artiste, album, année...) sont bien appliquées au fichier ogg.

C'est bien de la même chose qu'il s'agit. Ce qui fait que c'est encore plus bizarre !

La prochaine fois que ça me le fait, je mettrai le fichier.mp3 de côté et je te l'enverrai pour voir si tu reproduis le même comportement.

> ffmpeg vient bien de debian chez moi: 7:4.3.6-0+deb11u1

Même version ici.
